File: Python/Score Board/scoreBoard/GameEntry.py
class GameEntry:

    def __init__(self, name, score):
        self.name = name
        self.score = score

    def get_name(self):
        return self.name
    
    def get_score(self):
        return self.score
    
    def __str__(self):
        return f"{self.get_name()} -  {self.get_score()}"
    

File: Java/Score_Board/src/main/java/com/kucse/score_board/GameEntry.java
package com.kucse.score_board;

public class GameEntry {
    private String name;
    private int score;

    public GameEntry(String name, int score) {
        this.name = name;
        this.score = score;
    }

    public String getName() {
        return this.name;
    }

    public int getScore() {
        return this.score;
    }

    public String toString() {
        return "Name : " + this.getName() + "\nScore: " + this.getScore();
    }
}
